src/tool_application_support.cpp

```diff
@@ -27,38 +27,64 @@ namespace clangmetatool {
 
 namespace {
 
-std::string getExecutablePath(const std::string &argv0) {
+std::string getAbsoluteRealpath(const std::string &path) {
+  std::string absRealpath = path;
+  if (llvm::sys::path::is_relative) {
```

Comment:
I was wrong here, you don't need to make the path absolute. Realpath will take care of that